MAUI/MauiAppBasic/MSALClient/PCAWrapper.cs

```diff
@@ -30,14 +37,29 @@ public class PCAWrapper
         // private constructor for singleton
         private PCAWrapper()
         {
+            var assembly = Assembly.GetExecutingAssembly();
+            using var stream = assembly.GetManifestResourceStream("MauiAppBasic.appsettings.json");
+            AppConfiguration  = new ConfigurationBuilder()
+                .AddJsonStream(stream)
+                .Build();
+
             // Create PCA once. Make sure that all the config parameters below are passed
             PCA = PublicClientApplicationBuilder
-                                        .Create(AppConstants.ClientId)
+                                        .Create(AppConfiguration["AzureAd:ClientId"])
+                                        .WithTenantId(AppConfiguration["AzureAd:TenantId"])
                                         .WithExperimentalFeatures() // this is for upcoming logger
                                         .WithLogging(_logger)
                                         .WithRedirectUri(PlatformConfig.Instance.RedirectUri)
                                         .WithIosKeychainSecurityGroup("com.microsoft.adalcache")

```

Comment:
>com.microsoft.adalcache [](http://example.com/codeflow?start=71&length=23)

is there a need to use this "Adalcache" string? ideally, we should avoid adal at all costs